### PR TITLE
Infers type from css instead of using direct method

### LINKED ISSUE #4473: Heatmap CSS is not being correctly parsed
`wizard_properties.propertiesFromStyle` is returning a wrong/corrupt set of properties when used from Pecan to initialise a Heatmap wizard. Two properties return wrong:

- `image-filters` returns undefined because it's a function. I kind of fixed this already.
- `-torque-time-attribute` is set to `the_geom`, I'm not yet sure why.

cc @saleiva @javierarce

### PR DESCRIPTION
**Pull request by fdansv**
This doesn't entirely address the underlying problem, but it does resolve our problem with heatmaps so that we can worry about other problems for the time being.

Closes #4473 

@javierarce @saleiva

### CODE REVIEW

File: lib/assets/javascripts/cartodb/models/wizard.js [hunk 1]
```diff
@@ -524,7 +524,7 @@ cdb.admin.WizardProperties = cdb.core.Model.extend({
     var rules = parsed.getDefaultRules();
     if(parser.errors().length) return {};
     var props = {};
-    var t = this.get('type');
+    var t = this._getTypeFromCSS();
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/models/wizard.js]: It seems you forgot to pass the css in the params of the method.

File: lib/assets/javascripts/cartodb/models/wizard.js [hunk 2]
```diff
@@ -554,6 +554,16 @@ cdb.admin.WizardProperties = cdb.core.Model.extend({
...
   },
 
+  _getTypeFromCSS: function(css) {
+    if (css.indexOf("colorize-alpha") > -1) {
+      return "torque_heat";
+    }
+    else if (css.indexOf("torque-time-attribute") > -1) {
+      return "torque"
+    }
+    else return this.get('type');
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/models/wizard.js]: Add some curly braces around that last ```else```, to maintain the style we use in the conditionals.

File: lib/assets/javascripts/cartodb/models/wizard.js [hunk 3]
```diff
@@ -554,6 +554,16 @@ cdb.admin.WizardProperties = cdb.core.Model.extend({
     return {};
   },
 
+  _getTypeFromCSS: function(css) {
+    if (css.indexOf("colorize-alpha") > -1) {
+      return "torque_heat";
+    }
+    else if (css.indexOf("torque-time-attribute") > -1) {
+      return "torque"
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/models/wizard.js]: ```;``` :)

**Comment by fdansv** [comment]
@javierarce cabrón estás a todo! <3 Gracias por la CR en sábado ;)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5122/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5124/))

**Pull request merged**